src/Raven.Studio/typescript/components/pages/database/settings/documentExpiration/DocumentExpiration.tsx

```diff
@@ -293,7 +297,7 @@ const codeExample = `{
       "Set a timestamp in the @expires metadata property",
     "@metadata": {
       "@collection": "Foo",
-      "@expires": "2023-07-16T08:00:00.0000000Z"
+      "@expires": "${expireAt.toISOString()}"
```

Comment:
add `import moment from "moment";`
fix in other places
```suggestion
      "@expires": moment().add(1, "year").toISOString()
```

Reply:
done